pootle/apps/reports/views.py

```diff
@@ -568,6 +584,10 @@ def get_grouped_paid_words(scores, user=None, month=None):
             row['reviewed'] += reviewed_words
         row['score_delta'] += score.score_delta
 
+        suggested_words = score.get_suggested_words()
+        if suggested_words:
```

Comment:
Here you want to be explicit and use `if suggested_words is not None:` as you do above.